Python/pystate.c

```diff
@@ -569,11 +569,28 @@ _PyInterpreterState_Enable(_PyRuntimeState *runtime)
     return _PyStatus_OK();
 }
 
+#ifdef _Py_HAS_UNDEFINED_BEHAVIOR_SANITIZER
+#define RAW_ALIGN_PTR_OFFSET sizeof(void *)
+#endif
 
-static PyInterpreterState *
+static inline PyInterpreterState *
 alloc_interpreter(void)
 {
+#ifdef _Py_HAS_UNDEFINED_BEHAVIOR_SANITIZER
```

Comment:
I think that if we are going to include this code path, we should just perform the alignment unconditionally: 

* It's better to have our UBSan and other builds be as close as possible
* The extra space and checks are not a big deal in this case

Reply:
I'm ok with an unconditional custom alignment. Considering the size of an interpreter's size, it also makes sense to ease our life (I think the interpreter's size is about 40k or something along the lines, so adding a field shouldn't change much).